dev/buildtool/source_commands.py

```diff
@@ -138,6 +141,81 @@ def init_argparser(self, parser, defaults):
         )
 
 
+class TagBranchCommand(RepositoryCommandProcessor):
+    """Implements the tag_branch command."""
+
+    def __init__(self, factory, options):
+        """Implements CommandProcessor interface."""
+
+        self.__git = GitRunner(options)
+
+        all_names = list(SPINNAKER_RUNNABLE_REPOSITORY_NAMES)
+        all_names.extend(SPINNAKER_RUNNABLE_NON_CORE_REPOSITORY_NAMES)
+        all_names.extend(SPIN_REPOSITORY_NAMES)
+        super().__init__(factory, options, source_repository_names=all_names)
+
+    def _do_repository(self, repository):
+        """Implements RepositoryCommandProcessor interface."""
+        head_commit_id = self.__git.query_local_repository_commit_id(repository.git_dir)
+        logging.debug("%s HEAD commit: %s", repository.name, head_commit_id)
```

Comment:
include the branch name in the log message, here and below?

Reply:
Good idea, especially for when a GHA might run this.

Done in: https://github.com/spinnaker/buildtool/pull/190/commits/2ced9a75c5a496beb357387de79b1596ecb7cc5c
Example below: 
```
I 10:36:13.505 [Thread-4.2455274] fiat release-1.27.x branch HEAD commit: 52a8f5204dc12b693b8eac5ff6126759c119684a already tagged at: v1.28.5. Skipping.
```